src/mock/trip-event-list-data.js

```diff
@@ -0,0 +1,36 @@
+import dayjs from 'dayjs';
+import {generateTripOfferData} from '../mock/trip-offer-data.js';
+import {generateTripDestinationData} from '../mock/trip-destination-data.js';
```

Comment:
Эти импорты можно сократить, файлы ведь находятся в одной папке
```suggestion
import {generateTripOfferData} from './trip-offer-data.js';
import {generateTripDestinationData} from './trip-destination-data.js';
```